SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -156,7 +156,7 @@ void USpatialReceiver::OnAddComponent(Worker_AddComponentOp& Op)
 			Op.data.component_id, Op.entity_id);
 		return;
 	}
-	PendingAddComponents.Emplace(Op.entity_id, Op.data.component_id, MakeUnique<improbable::DynamicComponent>(Op.data));
+	PendingAddComponents.Emplace(Op.entity_id, Op.data.component_id, MakeUnique<SpatialGDK::DynamicComponent>(Op.data));
```

Comment:
```suggestion
	PendingAddComponents.Emplace(Op.entity_id, Op.data.component_id, MakeUnique<DynamicComponent>(Op.data));
```
This as well as other instances of `SpatialGDK` in this file seem unnecessary since we are `using namespace SpatialGDK`.

Reply:
Looks like your PR addresses this.